src/styles/blocks/header.scss

```diff
@@ -0,0 +1,22 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  box-sizing: border-box;
+  padding: 0 50px;
+  margin: auto;
+
+  @media screen and (min-width: 1024px) {
+    max-width: 1024px;
+  }
+
+  @media screen and (min-width: 1200px) {
+    max-width: 1200px;
+  }
```

Comment:
The media queries for `min-width: 1024px` and `min-width: 1200px` set `max-width` to the same values. This is redundant as the `max-width: 1200px;` will override the `max-width: 1024px;` for screens wider than 1200px, making the first media query unnecessary. Consider consolidating them into a single media query with the breakpoint at 1024px or removing the first one if 1200px is the intended breakpoint for your design.